### DISCUSSION TITLE
Who's hiring (Jan 2023)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#44541] [Ideas]
# What is this?
This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

If you hire someone through this job board, please reach out to me (hassan@vercel.com) and let me know so I can send you some swag :)

If you're a developer looking to post about themselves, use #44540.

*If you're looking for last month's discussion, see #43787. At least one company has [hired someone](https://twitter.com/Nutlope/status/1475929984174895105) as a result of it!*

**Comment by leif-cycle** [comment]
cycle ([cycle.de](https://cycle.de)) | OG of FrontEnd | remote or Hamburg/Frankfurt | ≥20 h/week | equity & 1k per Month to live life | 

We are looking for a _NextJS Master of Desaster_ to revolutionize the bike trading market at [cycle.de](https://cycle.de). Join us and ramp up [cycle.de](https://cycle.de) for the 2023 season.

**Teamwork & Techstack:** As a team we connect through Discord, Postman, Figma & Notion. At the FrontEnd are working with NextJS/vercel (well, captain obvious strikes again). The BackEnd API is well documented and has a performant & clean structure.

We build and bootstrap cycle.de & admin.cycle.de, the (Mail-)Server, CMS, bots, analytics and all other fancy stuff to be bloody good.

**YOU:** Besides being a little cool & crazy, you should be capable of coding strong and clean NextJS. Age, sex & origin doesn't matter, experience and proof of your work experience does.

HMU and we'll find out if we are a match. Just write us at ezclap@cycle.de. I'll reply asap⚡️

**Reply by rodinaleksandr0219** [reply]
Applied.
Please kindly check  your mail

**Comment by sivarcot** [comment]
Deestree |Full stack developer | EU Zone preferred | Freelance convertible to fulltime + equity | Dependent on location and XP | https://www.linkedin.com/hiring/jobs/3407402092/detail/

We are looking for top-notch full stack developers willing to change how food is sold in the B2B space. Remote freelance based in EU zone preferred, possible permanent position.

If you are passionate about technologies and you are a fast learner, you are the right person to join Deestree! Please contact: job-tech@deestree.com


**The company**

Deestree.com is a new wholesale food B2B marketplace built for independent suppliers to gain visibility. Suppliers are at last free to sell more easily at the price they want.

We are an early stage tech startup, growing super fast. We are onboarding our first customers and investors. We are building from scratch a new platform using the latest technologies and best development practices. We are obsessed with providing the best user experiences and simplicity to our users.

Deestree was founded by 3 domain experts: Ex-Amazon tech leader, multi startups founder, wholesale business owner.


**Your mission**

We will be working as freelance, optionally willing to move to a permanent position as a tech lead.

- Build features ground to top from the infrastructure (AWS, serverless, python, terraform etc) up to the UI (NextJS, Typescript, Bootstrap etc).
- Bring your experience to deliver a bug free platform to minimize customer impact.

**Your background**

We are not looking for language experts, we are looking for engineers who can solve problems.

- Engineering degree or equivalent
- 5+ years of experience as a software engineer
- An experience in front end development
- Strong experience in building scalable products
- Fluent in English

If you want to have more details, feel free to contact us: contact@deestree.com

**Reply by rodinaleksandr0219** [reply]
Hello @sivarcot 
Applied this position.

**Comment by Nsttt** [comment]
[Nexiona](https://nexiona.com/) | Frontend, Backend, Full-Stack, Systems | Remote (UTC +- 2) or Barcelona | Fulltime | 1 Friday off every month | Paid on-site team building activities every 6/8 weeks

We're currently looking for Frontend, Backend, Full-stack developers and Systems engineers. The positions are all Remote. 
Our Headquarters are located in Barcelona, we offer to our employees to work globally, in a flexible and collaborative environment.

The company is a software company with more than 10 years of experience in IoT and Industrial 4.0, we base all our business around MIIMETIQ, our leading product for the creation of IoT private platforms, industry 4.0 and smart buildings.

Some of our technologies are the following:
FE:
- React / NextJS.
- Typescript with focus on functional programming.
- React-query.
- TailwindCSS.
- fp-ts / io-ts
And we're slowly adopting the [T3 stack](https://github.com/t3-oss/create-t3-app) for some of our projects ❤️ .

BE:
- Mainly Python.
- Some microservices are on Rust.
- RabbitMQ.
- InfluxQL & MongoDB.
- MQTT, AMQP and other messaging protocols.

Our company also offer competitive compensation, 1 Friday off every month and paid on-site team building activities every 6/8 weeks in the beautiful Barcelona.

If you're interested on more info you can message me, or better send an Email to antuan.bordons@nexiona.com saying you come from Néstor.

**Reply by rammyblog** [reply]
@Nsttt Are you still hiring for the backend?

**Comment by cereallarceny** [comment]
[Raise Financial](https://raisefinancial.com) | Senior Full-Stack Engineer | Remote (North or South America) | Part or full-time | $30-50/hour contract

**Company and role**
Raise Financial is looking for a Next.js engineer to join our team as a part or full-time contractor. We're a fintech startup working to revolutionize the way people approach long-term investment. Interested candidates should know NextJS, specific NextJS 13, like the back of their hand. Work will be a combination of front-end and back-end, but we're also open to candidates showing preference to either end of the stack. This position is fully remote, but you must be located in North or South American due to timezone differences. We're prepared to work with you on an ongoing basis, or in time-limited contracts of 3, 6, or 12 months.

**Culture**
Well, we're a startup - you can't get much more relaxed than that! But don't worry, we've got a packed backlog and are ready to get **** done. We're working on the MVP of our initial product offering and hope to have something in users' hands within the next 6 months. The team is myself, and my co-founder, as well as a few other contractors that we work with for marketing and engineering. I've been a professional software developer for the last 15 years, and you would work directly with me on the project. We're flexible on pay as well, but would likely need to change the hours available to compensate. We don't care about college degrees and we don't do technical interviews - show us you can code and that you're a fun person to work with and we'll hire you over an MIT graduate any day of the week.

**Tech stack**
- React/NextJS
- Typescript
- Prisma
- Planetscale
- Vercel
- ChakraUI

If you're interested in a finding about more about the role - feel free to send an email with your resume to [patrick@raisefinancial.com](patrick@raisefinancial.com). Let's rock and roll!

**Reply by thejayhaykid** [reply]
Are you still looking for someone to fill this part time?

**Comment by josephskewes** [comment]
Alien Star Cloud (ID8R) | Full-Stack Developer / Cofounder | Remote (any location) | Part or Full-time | Equity Only (salary when funded)

**We are seeking a software developer to join our team (as cofounder/tech lead) to help serve the needs of millions of people who are being onboarded to web3.**

Sound interesting? Read on.

**About the startup (Alien Star Cloud)**

_We are building a design studio [(ID8R)](https://id8r.com/) for NFTs, enabling people to express themselves and show-off their treasured digital collectibles (NFTs) in exciting new ways._

- We incorporated the company in 2022.
- Two cofounders (AU, DE), you would in effect be our third.
- We have generated early revenue (non-recurring) with an MVP in the market.
- We see a path to growth, but we need a tech lead to continue developing our product and iterating with customer and market feedback.
- We raised a small F&F investment round which covers operational expenses.
- We believe there is a mass market opportunity for this product as we establish our roots and grow within an exciting new sector and industry.

**What we are looking for in a new team member**

- Someone passionate about the crypto / web3 / NFT space. Like us, you see this budding industry heading toward mainstream adoption, which means opportunity for new companies to seize significant market share.
- Someone who understands the grind of startups, which can often be a bumpy ride to finding product-market fit.
- Someone with experience (or a quick study) with the tech in the stack of our MVP: JavaScript / Next.js (on Vercel), AWS RDS w/ MySQL, Appsmith.
- Non-essential, but exposure to these may be handy: HTML Canvas, Three.js / React-three-fiber, Python / Stable Diffusion (+ training custom models).
- Someone with consistent availability (20+ hours/week) and ok with vesting equity as remuneration until funding/revenue that supports salaries.

If you have questions or you'd like to learn more, send me an email [joseph@alienstar.cloud](mailto:joseph@alienstar.cloud) or reach out by [Twitter DM](https://twitter.com/josephskewes).

I have a deck I can provide to those interested in high level vision / strategy.

**Comment by grant0417** [comment]
**Fig | Fullstack or Frontend Engineer | SF (Preferred) or Remote (US Timezone) | Full-time | [fig.io/hiring](https://fig.io/hiring)**

At Fig we are building the _next-generation command line_, building tools to increase developers productivity in the the terminal. Our current products are [autocomplete](https://github.com/withfig/autocomplete), [scripts](https://fig.io/user-manual/scripts), [dotfiles](https://fig.io/user-manual/dotfiles), and [access](https://fig.io/user-manual/access).

**Requirements:**

- Strong experience with our stack: Typescript, React/Next.js, Tailwind, Prisma, Express, Graphql
- Comfortable in the terminal
- Experience architecting and maintaining large typescript apps
- Great design skills

To apply either fill out an application here [fig.io/hiring](https://fig.io/hiring) or contact me directly at `grant <at> fig <dot> io`

**Reply by PrashantIndurkar** [reply]
Applied please check mail 🙏

**Reply by Taliss** [reply]
Hi @grant0417 , are you open to EU remote candidates?

**Comment by careersonextech** [comment]
### Freelance Software Engineer at One X Tech
One X Tech is on a mission to build the new digital economy by creating innovative digital solutions that put businesses at the digital frontier. As a software development company that focuses on building technological innovations for leading enterprises, One X Tech brings together an elite digital product team to create best-in-class products that scale across the world. One X Tech is based in Singapore and operates regionally in South East Asia. 

You can also find us on the web at [www.onextech.com](url)

**Location**: Remote
**Project duration**: Approx. 1-2 months with the option to extend
**Minimum commitment**: between 15-30 hours per week
**Timezone**: Singapore Time (GMT +8)
**Technologies**:
- ReactJS, Javascript, TypeScript
- High degree of fluency in JavaScript, with knowledge of modern techniques as well as core JS concepts
- Proficiency in frontend markup and styling - HTML and CSS.
- Able to design cross-browser and responsive pixel-perfect HTML. Be familiar with Material-UI.

If you're interested, send an email with your resume to careers@onextech.com

**Reply by AIEngKenn** [reply]
I might be interested in working with You, I live in Beijing (React Developer),
Tech Stack: Reactjs, Nextjs, GatsbyJS, TailwindCSS.

Currently looking for Remote Positions, Check my profile for more details.

Thanks

**Reply by PrashantIndurkar** [reply]
I'm interested.
i have sent the mail please check🙏

**Comment by aladrach** [comment]
Abnormal Security | Marketing Front End Developer | Remote - US/Canada | Full-time

Abnormal Security is defining the next generation of email security defense. Our platform uses machine learning and artificial intelligence to baseline communication content, user identity, and behavioral signals in real-time and at-scale in order to detect the abnormalities of email attacks.  Customers love us because we consistently detect and stop what everyone else in the market can’t -- advanced attacks that have never been seen before -- and we do so with beautiful user interfaces and best-in-industry customer support.

As a part of the Marketing Operations team, you will be responsible for maintaining and updating the Abnormal marketing-facing websites. You will work closely with our Creative team to implement new designs, as well as with our Content Marketing team to make both front end and CMS updates.

**Who you are:**
- You’re a motivated developer who is excited to learn new technologies and push your skills to the next level.
- You love a good challenge and have exceptional problem solving skills.
- You’re a team player who communicates well with your teammates.
- You have a great attention to detail, ensuring a precise, polished, finished product.

**What you’ll do:** 
- Develop efficient front end components and pages based on designs from Creative
- Optimize/test site components for consistent usability across all major browsers and viewports
- Maintain brand standards throughout website and related products
- Add/update content, create new pages, and optimize existing content as needed
- Make CMS schema updates as needed to add new sections or functionality

**Experience you’ll need:**
- 2+ years experience working in a fast-paced agile development environment (experience in marketing or agency work is preferred)
- Expert in HTML, CSS, and JavaScript
- Experience utilizing frameworks such as React, NextJS, GraphQL and Tailwind
- Experience in implementing designs from platforms such as Figma, Sketch, Photoshop, etc.
- Experience in SEO and mobile optimized development
- Experience in working within headless CMS platforms, preferably using GraphQL

If you're interested, send an email with your resume and portfolio to `aladrach AT abnormalsecurity DOT com`. We are seeking direct hires, so please no agencies at this time.

**Reply by lightofgiving** [reply]
I'm interested in this.
Sent out an email. Looking forward to hearing from you 🚀

**Comment by sharkey11** [comment]


**Reply by Phoenix-Alpha** [reply]
@sharkey11, is the position available to candidates in EU?

**Reply by sharkey11** [reply]
yes! @Phoenix-Alpha

**Comment by ashiva** [comment]


**Reply by r0valdez** [reply]
Hi Arash, I am interested in this position. I am trying to send an email to hello at freehumans.com but it is not found. What email should I send an email?

**Comment by zkwsk** [comment]


**Reply by Babbili** [reply]
Hello @zkwsk is there a site for the company please, thank you

**Comment by Metaphysics0** [comment]
### **Looking for a Full Stack Web Developer! (NYC Area) 🔥**

Check us out [here](https://www.cityhive.net/our-careers/), and
Message me at [ryanr@cityhive.net](mailto:ryanr@cityhive.net) if you want to work amongst some legends!

**Comment by IraES0116** [comment]
**EQUUS SOFTWARE | CLOUD SUPPORT ENGINEER | LISBON PORTUGAL | Full-time  | [LINK](https://apply.workable.com/equus-software/j/B07B2CE8F3/)**

Description
WHO WE ARE

Founded in 1999, Equus has continually been recognized by Inc. 5000 as one of the fastest growing companies in the United States. We continue to expand with offices in Denver, Dallas, Lisbon, London and Manila. Our relentless focus on creating innovative global mobility solutions has helped us become the market leader in our industry.

With over 300 clients around the world, we are committed to delivering the right solutions to our client’s problems. And we do this every day.

Our award-winning cloud-based technology has been recognized year-over-year for end-to-end functionality, efficiency savings and connected ecosystem. But we are more than just a technology company. We are experts in helping our clients get the right talent, in the right place, fast.

WHAT YOU DO

As a Cloud Support Engineer specializing in Azure and part of the IT Operations team, you will monitor, manage and maintain Equus’ client production environment. This role will heavily impact the growth and commitment of Equus Software to migrate clients from our existing cloud platform to Azure. You will be passionate about driving successful optimization within the environment. The opportunity to utilize the entire suite of Azure products within a global environment will allow you to develop SME level skills.

3-5 years monitoring cloud environments with an emphasis in Azure

Have a deep understanding of metric reporting – how to take information from Azure and create business cases to support environment optimization

Experience with Powershell scripting and SQL queries

Good communication skills

**No sponsorships available

NICE TO HAVE

Microsoft Azure certification

Database management experience

Strong Networking background

EDUCATION

Bachelor’s Degree in Information Technology or related field

Certifications in lieu of a degree will be considered

What is Equus?

Equus Software is a market leading Software Company specializing in Global Mobility Software. We provide award-winning service to our portfolio of high-profile global clients. Our team is known for providing creative software solutions to solve our clients’ complex business problems.

Why this is a great position:

Great salary and benefits – Highly competitive salaries, great bonuses for great contributors. We have enterprise-class benefits.

Great location – In the heart of Lisbon, with 290 days of sun a year and access to awesome restaurants and culture.

Great people – We only hire the best. That means you're working with the best.

Great culture – Nice office, relaxed and collaborative atmosphere, casual dress code.

Software is our business – This isn’t a department in a larger company. We’ve been developing software for 19 years, and we’re growing fast. You’ll be a part of a team that develops software quickly and effectively.

Effective development processes – We follow agile processes (kanban) while keeping focus on 2-month release plans. We also do continuous improvement.

Always learning something new – This is a large application that solves a lot of business problems for a lot of clients. You’ll develop a wide variety of skills, and you’ll probably become a subject matter expert in your first year.

Fostering innovation – Take advantage of Innovation Day with every release. Play with whatever technology you want, learn what you want, try out something new, and present it to the company the next day. Many innovations have made it into our products.

**Comment by martinzco** [comment]


**Reply by gyenabubakar** [reply]
Hi @martinzco, I’m interested in this. Would love to discuss more.

You can reach me via gyenabubakar@gmail.com

**Reply by christoph724** [reply]
Hi Martin, how do you do?
I'm a front-end developer with 8 years of experience, especially in Next.js.
I'm very interested and want to discuss this further.

Email: christophmartin724@gmail.com
Skype: live:.cid.f894bec847a3ed60

Thank you

**Reply by christoph724** [reply]
Hi Martin, how do you do?

I'm a Sr. frontend developer with over 8 years of rich experience, especially in Next.js.
Also, I've worked at agencies during my entire career so I have much experience working with agencies.
I do want to work with you and please let me know your contact information.

Email: christophmartin724@gmail.com
Skype: live:.cid.f894bec847a3ed60
Linkedin: [www.linkedin.com/in/christopher-martin-frontend-developer](http://www.linkedin.com/in/christopher-martin-frontend-developer)

I'm looking forward to hearing back from you.

Thank you,
Best regards,
Chris

**Comment by rodinaleksandr0219** [comment]
Hello Martin. How are you doing?

I am interested in this opportunity so would like to discuss more
details with you.

Regards,

**Comment by GreatWorldDev** [comment]
Hi, I am a senior frontend developer. 
https://software-mars.vercel.app

**Comment by ivansf** [comment]
Full-stack developer | Remote: PST/MST/EST location | 3 months contract

Looking for full stack developers with extensive experience in Web3 (EVM + Hedera) mints and gaming mechanics. Message me if interested.

**Reply by GreatWorldDev** [reply]
@ivansf  I am a full-stack web3 developer. 
https://software-mars.vercel.app
Please contact me.

**Reply by WebVA** [reply]
@ivansf I have rich expertise in Full Stack Web & Blockchain development.
Please check my github and if you would like to get in touch to discuss further then please do so at your convenience.
Kind regards
